docs/source/import_export/main.rst

```diff
@@ -9,30 +9,29 @@ Export
 
 Use ``verdi export create`` to export a selection of nodes, codes, groups, users, or computers.
 
-* **Selection:** Nodes can be selected via identifier (PK, UUID, label) or by exporting predefined groups of nodes.
-* **Augmentation:** By default, the export function augments the selection of nodes by their parents in order to preserve the provenance.
-  For calculation nodes, the direct outputs are also added.
-  By default, comments and logs are also added for each exported node.
-  These may be excluded using the proper flags.
-  There are several other options to export specific parts of the provenance graph, starting from the selection.
-  For more information about this please read the :ref:`corresponding subsection<consistency>` of the :ref:`Provenance section<concepts_provenance>`.
+See ``verdi export create -h`` for a full list of available options.
+
+* **Selection:** Nodes can be selected via identifier (PK, UUID, label).
+  It is often convenient to first put the nodes you would like to export into a :py:class:`~aiida.orm.Group` and then to export the group using ``verdi export create -G <group-name>``.
+* **Augmentation:** By default, the export function automatically augments your selection of nodes by their logical and data provenance (and calculations by their direct outputs).
+  You can customize this behavior using command line options.
```

Comment:
If the default behavior is still to export comments and logs and you can disable this, then I would maybe keep the line clarifying this (maybe not in the **augmentation** section but in the **file content** section below? unless you think it is obvious that this "metadata" would be included) because the data consistency section only discusses what nodes are included or not, and doesn't deal with what kind of information gets exported for each node.

Reply:
I agree, this should be left in as a comment, since it may result in a lot of extra exported data if one has a lot of Logs (or really use Comments a lot - in which case one would probably _want_ to export them... :sweat_smile: )

Reply:
My thinking was that users should expect this to be included, and that they will see the cli option when they run the `-h` command:
```
 --include-logs / --exclude-logs
                                  Include or exclude logs for node(s) in
                                  export.  [default: True]
  --include-comments / --exclude-comments
                                  Include or exclude comments for node(s) in
                                  export. (Will also export extra users who
                                  commented).  [default: True]
```

Reply:
We might want to mention if certain data is *not* included by default - to my knowledge it's only the `Authinfo` of the computers (?).

Reply:
Since people shouldn't directly interact with `Authinfo` anyway (I believe?), I don't see that it needs to be written here - the more technical users can in any case refer to the `metadata.json` example to see what is and what isn't exported?

Reply:
I agree.